<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ChooseImage">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ChooseImage</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string>                  Image1</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string>                  Image2</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>                 Image3</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>               Image4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PreProcess</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConvertToGrey">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>灰度处理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GreyHistogram">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>灰度直方图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GreyBalance">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>灰度均衡</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GradientSharpening">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>梯度锐化</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>260</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>边缘检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Roberts">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>300</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Roberts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sobel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>330</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sobel &amp;&amp; Schar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Laplace">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>360</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Laplace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>390</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Prewitt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="canny">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>420</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Canny</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1151</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
